theme/common.less

```diff
@@ -181,6 +181,44 @@ div.simframe > iframe {
 }
 
 /* Menu */
+#menubar #accessibleMenu {
+    position: absolute;
+    z-index: 1001;
+    top: -20em;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    list-style-type: none;
+
+    .ui.item.link {
+        position: absolute;
+        width: 100%;
+        color: rgba(255, 255, 255, 0.9);
+    }
+
+    .ui.item.link:focus {
+        top:20em;
+    }
+
+    .ui.item.link:hover {
+        top:20em;
+    }
+}
+
+@media only screen and (max-width: @largestTabletScreen) {
+    #menubar .ui.menu {
```

Comment:
What's this? Why is it here. 

If there is a reason to add this media query, it should be with the rest of the media queries in the bottom of the file. and please add a comment above it.